.gitignore

```diff
@@ -1,24 +0,0 @@
-*~
```

Comment:
Same here... I'm not sure how you deleted these files from your working copy but you should figure that out so it doesn't happen again...